server/controllers/rpaFrameworkCommandsController.js

```diff
@@ -0,0 +1,51 @@
+const rpaFrameworkCommands = require('../models/rpaFrameworkCommands');
```

Comment:
This is not used

Reply:
I put it in for demonstrating purposes but I can remove it now :)